Makefile

```diff
@@ -336,4 +384,8 @@ install-deb: install-common install-systemd install-systemd-dropins
 	mkdir -p $(DESTDIR)/etc/systemd/system/
 	install -m 0644 vm-systemd/haveged.service  $(DESTDIR)/etc/systemd/system/
 
-install-vm: install-rh install-common
+install-corevm: install-rh install-common install-systemd install-sysvinit install-systemd-dropins install-networking
+
+install-netvm: install-systemd-networking-dropins install-networkmanager install-netvm
```

Comment:
`install-netvm` shouldn't be as dependency of itself...

Reply:
Commited